CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ All notable changes to this project will be documented in this file.
 ## [0.10.1] - 2019-09-30
 - Upgrade to Airflow 1.10.5
 - Kubernetes 1.14+ support
-- Improved metrics in Grafana and Orbit
+- Improved metrics in Grafana and the UI
```

Comment:
not sure that it's good idea to edit old releases in CHANGELGO
@ash wdyt?
better to create new section 0.12 or so and add 1 line.

Reply:
Yea went back and forth on that too. Figured as we move forward, orbit would loose meaning for folks inside and outside the org, so this would make it clearer. Open to change if others feel otherwise.